assets/stylesheets/style.scss

```diff
@@ -224,8 +255,11 @@
 
     // WCC Components
     @import '../../client/components/shipping/packages/style';
-    @import '../../client/components/indicators/style';
-    @import '../../client/components/text/style';
+	@import '../../client/components/indicators/style';
```

Comment:
@DanReyLop I see you switched to tabs for these lines

@kellychoffman - unlike our JS with tabs, we seem to use spaces in our SCSS, including this file - any idea why?